consul/util.go

```diff
@@ -125,6 +130,11 @@ func isConsulServer(m serf.Member) (bool, *serverParts) {
 
 	datacenter := m.Tags["dc"]
 	_, bootstrap := m.Tags["bootstrap"]
+	var disabled uint64 = 0
+	_, disabledStr := m.Tags["disabled"]
```

Comment:
The *only* reason I pushed this into a tag was it seemed conceivable that we'd want to support a "consul server slow-start" where a server would come up in a disabled state.  100% okay with removing support for this.